src/pages/Main/Main.js

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { wordsActions } from '../../actions/wordsActions';
+import { DispatchWordsContext, StateWordsContext } from '../../context/WordsContext';
+import { getHundredWords } from '../../api/getHundredWords';
+import { PagesForm, 
+         SearchForm, 
+         Subscription, 
+         WordsOnMainPage, 
+} from '../../components/index';
+import { Tabs, Table } from '../../common/index';
+
+import './main.scss';
+
+
+const Main = () => {
+  const [activeTabIndex, setActiveTabIndex] = useState(1);
```

Comment:
Try to avoid 'magic numbers'. Please describe it as a 'defaultTabId', for example, and use like `useState(defaultTabId);`

Reply:
Done